### PR TITLE
feat: added search endpoint

### PR DESCRIPTION
**Pull request by codingholt**
<details>
  <summary>Added this search endpoint to the wrapper.
</summary>
 https://api.coingecko.com/api/v3/search?query=<search query goes here>
</details>

Returns: List of coins, categories and markets matching search query ordered by market cap

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,13 +0,0 @@
-{
```
Comment (tomholford) [inline tsconfig.json:1]: @codingholt any reason the tsconfig was removed?

**Comment by tomholford** [comment]
@samuraitruong Can you merge and publish this to `npm`?

File: tsconfig.json (hunk 1 above)
Comment (codingholt) [reply tsconfig.json:1]: Um no, I don't think so. I must've done it by accident.

Comment (codingholt) [reply tsconfig.json]: thanks for pointing it out!

**Comment by samuraitruong** [comment]
@tomholford  I merged #20. we we still need this?

**Comment by tomholford** [comment]
> @tomholford I merged #20. we we still need this?

#20 had the same functionality as this, but a couple more clean up tweaks. So, okay to close this PR

**Comment by codingholt** [comment]
closing as #20 resolves the same issue and is already merged

**Pull request closed without merging**